app/config.js

```diff
@@ -1,17 +1,18 @@
-var join = require('path').join;
-var config = module.exports;
-var PROD = process.env.ENV === 'prod';
+const join = require('path').join;
 
-var node_modules = join(__dirname, '../node_modules');
 
+const config = module.exports;
```

Comment:
I don't know if I'm missing something but I'm wondering if all these `config.something = {}` could be replaced with a single big:

```javascript
module.exports = {
  express: {
    // ...
  },
  middleware = [
    // ...
  ],
  // etc...
}
```

Reply:
But how would that improve things?

Reply:
I actually prefer this style (not my idea, I got it from someone else), because you always reference config variables in the same style `config.foo.bar`, even in the definition.

Reply:
I suppose in mind the reason why I was saying it was because I thought about it as a big JSON config file - but you're right change doesn't probably give us much 👍 (and the way it is clear to me)